src/docs/user/references/ScriptLanguageSupport.adoc

```diff
@@ -396,6 +396,50 @@ Or another script binding (for example, USERSPACE):
 my $USERSPACE= $ENV{"USERSPACE"};
 ----
 
+=== PHP
+
+The PHP script engine features the following behaviors:
+
+* Script bindings are translated as environment variables.
+* Workflow variables are un-modifiable except by using a post-script written in another language such as the `update_variables_from_file` script (see <<_bash_and_shell,Bash script language>> for further explanations).
```

Comment:
immutable is more technical than un-modifiable

Reply:
except by using a post-script written in another language such as... => unless a post-script, written in another language, is used such as ...

Reply:
in this context immutable would be difficult to understand. I could also say "Workflow variables cannot be modified by a PHP script except"